pom.xml

```diff
@@ -70,7 +70,7 @@
     <!-- do not forget to also upgrade hamcrest library with junit -->
     <cs.junit.version>4.12</cs.junit.version>
     <cs.hamcrest.version>1.3</cs.hamcrest.version>
-    <cs.bcprov.version>1.46</cs.bcprov.version>
+    <cs.bcprov.version>1.54</cs.bcprov.version>
```

Comment:
it appears that [1.55](http://search.maven.org/#artifactdetails%7Corg.bouncycastle%7Cbcprov-jdk14%7C1.55%7Cjar) has been released.  Seems like a good idea to use the latest.